frontend/pulse3d/src/components/basicWidgets/CircularProgressWithLabel.js

```diff
@@ -1,10 +1,10 @@
 import CircularProgress from "@mui/material/CircularProgress";
 import Box from "@mui/material/Box";
 import Typography from "@mui/material/Typography";
-export default function CircularProgressWithLabel(props) {
+export default function CircularProgressWithLabel({ value, colorOfTextLabel = "black" }) {
```

Comment:
line 22 will need to be `props.value` -> `value`

Reply:
I was testing and just found this as well